APIs/SentinelHub/Overview/ProcessingUnit.qmd

```diff
@@ -80,7 +80,7 @@ apply with the following exceptions:
     factor of 1/3. Thus, three times more data can be processed
     comparing to process API for the same amount of PUs.
 -   \*\* When data is delivered to a bucket in other region within the
-    same system (i.e. Copernicus Data Space Ecosystem, AWS) there is additional cost of 0.03 PU per
+    same system (i.e. CDAS, AWS) there is additional cost of 0.03 PU per
```

Comment:
CDAS? is that an intended change?

Reply:
No it was not intended
It seems like the old changes were accepted by mistake that cause acceptance of old issues. However, updated information are already existing in the current production version